Site/SiteApplication.php

```diff
@@ -448,26 +431,17 @@ protected function setUpFileErrorHandling(SiteConfigModule $config)
 	 */
 	protected function setUpSentryErrorHandling(SiteConfigModule $config)
 	{
-		$client = new Raven_Client(
-			$config->sentry->dsn,
-			array(
-				// Default breadcrumb handlers override the error_reporting
-				// settings, so we disable them
-				'install_default_breadcrumb_handlers' => false,
-				// Catch fatal errors with Sentry.
-				'install_shutdown_handler' => true,
-				'environment' => $config->sentry->environment,
-			)
-		);
-
-		SwatException::addLogger(new SiteSentryExceptionLogger($client));
-		SwatError::addLogger(new SiteSentryErrorLogger($client));
-
-		// Add fatal error handling.
-		$error_handler = new Raven_ErrorHandler($client, false, null);
-		$error_handler->registerShutdownFunction();
-
-		$this->sentry_client = $client;
+		\Sentry\Init([
+			'dsn' => $config->sentry->dsn,
+			'environment' => $config->sentry->environment,
+			'default_integrations' => false,
+			'integrations' => [
+				new \Sentry\Integration\FatalErrorListenerIntegration()
```

Comment:
It seems like the error handler we were using previously got replaced by this integrations feature. Since before we were only using fatal error handling, I disabled the others and only allowed fatal: https://docs.sentry.io/platforms/php/integrations/